src/common/utils.cpp

```diff
@@ -70,14 +70,14 @@ void ShowDump(const void* buffer, size_t length)
 
 		if( (i%16) == 15 )
 		{
-			ShowDebug("%03X %s  %s\n", i/16, hex, ascii);
+			ShowDebug("%03lu %s  %s\n", i/16, hex, ascii);
```

Comment:
Depending on x86 or x64 should be %03X or %03llX.
Definitely requires the X though for hex representation.
Additionally size_t is not unsigned.

Reply:
fixed by using PRIXPTR